.gitignore

```diff
@@ -20,3 +20,7 @@ earth_enterprise/tutorial/FusionTutorial/KML
 /*.creator.user
 /*.files
 /*.includes
+
+# properties files
+*.properties
```

Comment:
Sorry, we can't blanket ignore all ".properties" files. Plus, this is a common type and it might frustrate devs who need to add a new file and can't figure out why  I've found two legitimate ones that are checked in. You'll need to figure out how to more specifically filter out the properties files you don't want included.
```
$ find . | grep properties
./src/server/wsgi/conf/postgres.properties
...
./rpms/gradle/wrapper/gradle-wrapper.properties
```

Reply:
We can just ignore this specific one in rpms/build.properties then...or call it something else, like I originally did with .buildinfo.

Reply:
Actually I found there is a .gitignore in the rpms/ directory so I could move it to there...

Reply:
Unfortunately, that won't quite do the trick since one of the valid properties files is under the rpms directory. "./rpms/gradle/wrapper/gradle-wrapper.properties" Probably best to just ignore the specific build.properties file instead of all of them.